.github/actions/e2e_test/action.yml

```diff
@@ -50,26 +53,14 @@ inputs:
   awsOpenSearchPwd:
     description: "AWS OpenSearch Password to upload the benchmark results."
     required: false
-  azureSubscription:
-    description: "The Azure subscription ID to deploy Constellation in."
-    required: false
-  azureTenant:
-    description: "The Azure tenant ID to deploy Constellation in."
-    required: false
-  azureClientID:
-    description: "The client ID of the application registration created for Constellation in Azure."
-    required: false
-  azureClientSecret:
-    description: "The client secret value of the used secret"
-    required: false
-  azureUserAssignedIdentity:
-    description: "The Azure user assigned identity to use for Constellation."
-    required: false
-  azureResourceGroup:
-    description: "The resource group to use"
-    required: false
+  azureClusterCredentials:
+    description: "Azure credentials authorized to create a Constellation cluster."
+    required: true
+  azureIAMCredentials:
```

Comment:
Maybe `azureCICredentials` works better as all credentials are "IAM" credentials, so it is not totally obvious that we mean IAM creation.

Reply:
s.a.